packages/pn-personafisica-webapp/src/api/external-registries/__test__/External-registries.api.test.ts

```diff
@@ -1,18 +1,24 @@
 import MockAdapter from 'axios-mock-adapter';
 
-import { mockAuthentication} from "../../../redux/auth/__test__/test-utils";
+import { mockAuthentication } from "../../../redux/auth/__test__/test-utils";
 import { ExternalRegistriesAPI } from "../External-registries.api";
 import { apiClient } from "../../apiClients";
 import { GET_ALL_ACTIVATED_PARTIES } from "../external-registries-routes";
+import { mockApi } from '../../../__test__/test-utils';
 
 describe('ExternalRegistries API tests', () => {
+  let mock: MockAdapter;
   mockAuthentication();
-  test('getAllActivatedParties 200', async () => {
-    const mock = new MockAdapter(apiClient);
-    mock.onGet(GET_ALL_ACTIVATED_PARTIES(undefined)).reply(200, []);
+  afterEach(() => {
+    if (mock) {
+      mock.restore();
+      mock.reset();
+    }
+  });
+
+  it('getAllActivatedParties 200', async () => {
+    mockApi(apiClient, 'GET', GET_ALL_ACTIVATED_PARTIES(undefined), 200, undefined, []);
```

Comment:
as before you are not setting mock istance. check all tests

Reply:
changes made into: [430db73](https://github.com/pagopa/pn-frontend/pull/938/commits/430db73dd4e6ee0215846db2b30db4717bdd1763)